File: 프로그래머스/unrated/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index):
    alphabet = []
    answer=''
    for i in range(97,123):
        if chr(i) not in skip:
            alphabet.append(chr(i))
    for c in s:
        idx = alphabet.index(c)+index
        if idx >= len(alphabet):
            idx %= len(alphabet)
        answer+=alphabet[idx]
    return answer

File: 프로그래머스/unrated/155652. 둘만의 암호/둘만의 암호.js
function solution(s, skip, index) {
    const alphabet = []
    for(let i=97; i <= 122; i++){
        let asc = String.fromCharCode(i)
        if(skip.includes(asc) === false) alphabet.push(asc);
    }
    
    let answer = ''
    
    for(c of s){
        let idx = alphabet.indexOf(c)+index
        if(idx >= alphabet.length) idx %= alphabet.length;
        answer+=alphabet[idx]
    }
    return answer
}